lib/perl/Genome/VariantReporting/Process/CreateReport.pm

```diff
@@ -83,6 +84,38 @@ sub create_disk_allocation {
     return $rv;
 }
 
-
+sub _compare_output_files {
+    my ($self, $other_process) = @_;
+
+    my %diffs;
```

Comment:
It seems more intuitive to me for `%diffs` to be keyed off the subpath (common path) of the two paths, e.g. `File::Spec->abs2rel($other_file, $other_output_dir)` rather than the absolute paths of either `$file` or `$other_file`.  Alternatively you might just store them in an array if they keys don't really matter at all.